### ISSUE TITLE
small variance explained by PRScs

### ISSUE THREAD

**Issue by tycui** [getian107/PRScs#42]
Dear Tian,

Thank you for providing such an amazing method. I'm testing different PRS methods on NMR metabolomics phenotype (HDL cholesterol) from UKBiobank. I separate the dataset (after some QC steps on SNPs, ~200k SNPs are left) into training (~100k individuals) and testing (~20k individuals), where the GWAS summary statistics are estimated on the training set only with plink -assoc (without normalizing the phenotype). Then I run PRScs with GWAS from the training set, UKB eur LD reference panel, and the test data bim files. I compute the PRS score with plink --score 2 4 6 sum, and then I  sum over the scores over different chromosomes. 

I evaluate the performance with the variance of phenotype explained by the PRS. However, I only get an R^2 ~0.009 for PRScs on the test set (phi auto and other hyper-parameters are set to default) but the PRS constructed with Lasso (plink --lasso) on the training set can have R^2 ~0.06 on the test set. I'm wondering if such results are expected or something might be wrong or some hyper-parameters can be improved in my experiments with PRScs. 

Thanks in advance!

Best,
Tianyu

**Comment by getian107** [comment]
Hi Tianyu- Did you use the genotyped SNPs or imputed data from the UK Biobank? 200K seems a small number to start with. PRScs uses HapMap3 variants to build PRS so the actual number of variants that went into the PRS may be very small. Meanwhile lasso used all the available SNPs in the dataset to build PRS, which might explain the performance difference. Usually with a well-imputed dataset such as UK Biobank, the final number of SNPs utilized by PRScs is usually around a million.

**Comment by tycui** [comment]
Hi Tian, thanks for the quick reply. I used the imputed data from UKB and Lasso has the same number of SNPs as PRScs. 

I tried PRScs again but using the GWAS source in your paper, instead of estimating the GWAS on the training set, which gave me R^2 ~0.07 on the test set. So I guess there is something wrong with the plink GWAS estimation. Do we need to normalize the phenotype and genotype in GWAS to construct PRScs? Actually, I'm quite new to constructing GWAS with plink. Do you think a simple run `plink --assoc` is sufficient for PRScs?

**Comment by getian107** [comment]
Hi Tianyu- Using plink --assoc for GWAS should be fine. After you get the plink output, did you reformat the summary statistics into the format that PRScs requires (i.e., 5 columns with appropriate column header)?

**Comment by tycui** [comment]
Hi Tian. Actually, I have reformated the summary statistics from plink. It's wired that most of the significant SNPs in GWAS from the training set are not significant in the external GWAS source and vice versa, but I can try to figure it out. Thank you a lot!
